bin/arm64_env.sh

```diff
@@ -17,4 +17,11 @@ export supportRulesengine=nexus3.edgexfoundry.org:10004/docker-support-rulesengi
 export exportClient=nexus3.edgexfoundry.org:10004/docker-export-client-go-arm64:1.0.0
 export exportDistro=nexus3.edgexfoundry.org:10004/docker-export-distro-go-arm64:1.0.0
 export deviceVirtual=nexus3.edgexfoundry.org:10004/docker-device-virtual-arm64:0.6.0
+
+export vault=nexus3.edgexfoundry.org:10004/docker-edgex-vault:0.2.0
+export vaultWorker=nexus3.edgexfoundry.org:10004/docker-edgex-vault-worker:latest
+export kongdb=postgres:9.5
+export kong=kong:0.13.0
+export edgexProxy=nexus3.edgexfoundry.org:10004/docker-edgex-proxy-go:latest
+
```

Comment:
For Lines 21 through 25 in this file we need to point at the arm64 versions of these services.  The kong and postgres images may be multi architecture, I'm not sure.  But the EdgeX images will have `-arm64` in the image name.

Reply:
`nexus3.edgexfoundry.org:10004/docker-edgex-vault-worker-go-arm64:0.3.0`
`nexus3.edgexfoundry.org:10004/docker-edgex-vault-arm64:0.3.0`
`nexus3.edgexfoundry.org:10004/docker-edgex-proxy-go-arm64:0.2.1`

I got those image names and tags from here https://nexus3.edgexfoundry.org/#browse/browse:docker.staging
If you see an image tag that should be there but is not reconciled with the security repo master tag please let me know.

Reply:
Updated the file for arm.

Reply:
I'm not seeing those changes in the file @tingyuz , I may be missing it though.

Reply:
my fault... just recheck in the file and trigger the build.